test/commands/force/apex/log/list.test.ts

```diff
@@ -77,33 +77,38 @@ const cleanResult = {
       Status: 'Successful',
     },
   ],
+  warnings: [],
 };
 
 const logRecords = [rawLogResult.result[0], rawLogResult.result[1]];
 
 describe('force:apex:log:list', () => {
+  before(() => {
+    process.exitCode = 0;
+  });
+
   test
     .withOrg({ username: 'test@username.com' }, true)
     .stub(LogService.prototype, 'getLogRecords', () => logRecords)
     .stdout()
-    .command(['force:apex:log:list', '--targetusername', 'test@username.com'])
+    .command(['force:apex:log:list', '-o', 'test@username.com'])
     .it('runs default command with cleaned table output', (ctx) => {
-      let r =
-        'APPLICATION  DURATION (MS)  ID                   LOCATION  SIZE (B)  LOG USER    OPERATION  REQUEST  START TIME                STATUS          ';
-      r +=
-        '\n───────────  ─────────────  ───────────────────  ────────  ────────  ──────────  ─────────  ───────  ────────────────────────  ────────────────\nUnknown';
-      r +=
-        '      75             07L5tgg0005PGdTnEAL  Unknown   450       Test User   API        API      2020-10-13T05:39:43+0000  Assertion Failed\nUnknown';
-      r +=
-        '      75             07L5tgg0005PGdTnFPL  Unknown   450       Test User2  API        API      2020-10-13T05:39:43+0000  Successful      \n\n';
-      expect(ctx.stdout).to.equal(r);
+      expect(ctx.stdout).to.include(
```

Comment:
eww.  I think I'd rather assert that the correct args are passed to `this.table` instead of writing an integration test for cliUx